src/main.js

```diff
@@ -7,60 +7,73 @@ var ideaForm = document.querySelector('form');
 var saveButton = document.querySelector('.idea-form-button');
 var hamburgerIcon = document.querySelector('.hamburger-icon');
 var menuCloseIcon = document.querySelector('.menu-close-icon');
-var menuDropDown = document.querySelector('.menu-dropdown')
-var blanket = document.querySelector('.blanket')
+var menuDropDown = document.querySelector('.menu-dropdown');
+var blanket = document.querySelector('.blanket');
+var starredIdeasButton = document.querySelector('.show-starred-ideas');
+var searchBar = document.querySelector('.search-input');
 
+var favoritedIdeas = [];
```

Comment:
Is this necessary anymore or can we use it to our advantage to clear up some of the repeat code in the bottom?